### ISSUE TITLE
"Hakuna Matata" offensive phrase

### ISSUE THREAD

**Issue by ic-it** [amwmedia/eslint-plugin-woke#12]
https://github.com/amwmedia/eslint-plugin-woke/blob/3efe86a109b9e29207449a47bd54f53014885c5c/lib/categories/racism.js#L104

I think, you have an incomplete list. Please, add this phrase for me:
https://youtu.be/ZOSTvOlL-hw

**Comment by amwmedia** [comment]
Certainly, I don't expect the list to be complete. Please feel free to submit PR requests for anything I've missed.
